usage_counter_test.go

```diff
@@ -246,15 +238,13 @@ func TestUsageCounterTwoIntervals(t *testing.T) {
 func TestUsageCounterTwoIntervalsSinglePoll(t *testing.T) {
 	req := require.New(t)
 
-	intervalAgeThreshold = 0
-	defer resetAgeThreshold()
-
 	closeNotify := make(chan struct{})
 	defer close(closeNotify)
 
 	interval := time.Second
 
-	registry := NewUsageRegistry("test", nil, closeNotify).(*usageRegistryImpl)
+	config := DefaultUsageRegistryConfig("test", closeNotify)
+	registry := NewUsageRegistry(config).(*usageRegistryImpl)
```

Comment:
it'd be good to see tests for the minimums too (maybe they are here just not shown in the diffs?)